pages/common/7za.md

```diff
@@ -0,0 +1,19 @@
+# 7za
+
+> A file archiver with highest compression ratio
+
+- package and compress directory or file
+
+`7za a {{compressed.7z}} {{directory_or_file}}`
+
+- decompress an existing 7z file
+
+`7za x {{compressed.7z}}`
+
+- compress to zip format
+
+`7za a  -tzip {{compressed.zip}} {{directory_or_file}}`
+
+- create multipart 7zip file
+
+`7za -v{{size}}{{[b|k|m|g]}} {{compressed.7z}} {{directory_or_file}}`
```

Comment:
so if I understood correctly, `7xa -v12g a.7z here` should make a multipart zip file with each part having at most 12 gigs, of what, of the directory? or of the 7z file? will the 7z file be split into several smaller files and stored within the directory?

This one is quite unclear. Try something like:

```
- create a multipart 7zip file; part_size being size plus factor (b for byte,k for kilobyte)
7za -v{{part_size}} ....the rest here
```

It's easy when to read when you understand the command, the point is that most people won't, and thus it has to be written for them instead